models/post-comment-model.js

```diff
@@ -0,0 +1,43 @@
+//REQUIRES
+const db = require("../db/connection.js");
+
+//FUNCTION
+function newComment(article_id, postingComment) {
+    let dateObj = new Date();
+    if (!postingComment.body) {
+        return Promise.reject({
+            status: 400,
+            msg: `Nothing to post!`
+        })
+    }
+    if (!postingComment.username) {
+        return Promise.reject({
+            status: 400,
+            msg: `No anonymous posting allowed, please log in`,
+      });
+    }
+    if (Object.keys(postingComment).length !==2) {
+        return Promise.reject({
+            status: 400,
```

Comment:
This logic for `.length !== 2` could be removed, as long as it does have the _body_ and _username_ it's good to go!